### ISSUE TITLE
I Fixed the bots :D

### ISSUE THREAD

**Issue by mayed505** [davidmann4/ogar-feeder-bot#556]
im the first one to fix my own bots :dash: 
https://gyazo.com/48f6db7da3eb58990dee729fa2a73d87

**Comment by ghost** [comment]
I don't understand why u just scam ,

**Comment by mayed505** [comment]
@problemsofworld im not? i can proof it if you want....

**Comment by mayed505** [comment]
@problemsofworld the fix is pretty easy it has to do with the userscript.

**Comment by ghost** [comment]
then just show bots , so other's can get hope to fix there bots

**Comment by mayed505** [comment]
@problemsofworld im not releasing it so agario doesnt patch it again

**Comment by Nick8fr32** [comment]
U dont have a fix cause ur bots Dont spawn in

**Comment by mayed505** [comment]
@Nick8fr32 how do the heck even you know?

**Comment by Nick8fr32** [comment]
because i fixed the server thing to but i cant connect because agario-client dont work anymore

**Comment by ghost** [comment]
no one has fix yet .

**Comment by mayed505** [comment]
raga.pw and agarbot.ovh?

**Comment by ghost** [comment]
rip

**Comment by mayed505** [comment]
im jsut chillling play slither.io with raga bots :D

**Comment by yario-o** [comment]
the bots can't spawning

**Comment by ghost** [comment]
shlither.io is shit I don't  like games thou lol

**Comment by mayed505** [comment]
worms.raga.pw

**Comment by yario-o** [comment]
but there is a fix for the ip connect

**Comment by Nick8fr32** [comment]
agarbot.ovh dont work

**Comment by Nick8fr32** [comment]
yes @yario-o but it dont help when agario-client dont work

**Comment by yario-o** [comment]
yes

**Comment by yario-o** [comment]
The error when bot try to spawn is : getaddrinfo ENOTFOUND

**Comment by ghost** [comment]
sad only pro coders can fix bots

**Comment by yario-o** [comment]
But it will take long time

**Comment by yario-o** [comment]
because the agario protocol had totally changed

**Comment by ghost** [comment]
I  hate waiting  .

**Comment by Nick8fr32** [comment]
only a week i think XD we can also fix it if we work hard

**Comment by yario-o** [comment]
yes

**Comment by Nick8fr32** [comment]
we need to fix agario-client -_- and its [closed](https://github.com/pulviscriptor/agario-client/issues/193) now

**Comment by yario-o** [comment]
yes

**Comment by yario-o** [comment]
But we can modify it

**Comment by Nick8fr32** [comment]
its gonna take alot of time o.O

**Comment by yario-o** [comment]
yes

**Comment by yario-o** [comment]
but it's the only way

**Comment by Nick8fr32** [comment]
i think i am just gonna start on bots for slither.io

**Comment by yario-o** [comment]
okay

**Comment by mayed505** [comment]
@Nick8fr32 i dont think you would be sucess with that since raga.pw pays devs thousands of dollars in research.

**Comment by mayed505** [comment]
if i was rich as hell i would buy raga company :D

**Comment by Nick8fr32** [comment]
@mayed505 yea i dont know but i am gonna give it a try

**Comment by mayed505** [comment]
you atleast need an api first :/

**Comment by yario-o** [comment]
@mayed505 he can do this
slither.io is a new game and the bots are not patch

**Comment by mayed505** [comment]
but you will need a server connect just like the skeleton of agario-client thats what i meant :D

**Comment by yario-o** [comment]
@mayed505 There is also private server in slither.io and it don't take a long time

**Comment by Nick8fr32** [comment]
https://github.com/ClitherProject/Slither.io-Protocol

**Comment by mayed505** [comment]
@Nick8fr32 clever....

**Comment by Nick8fr32** [comment]
XD @mayed505

**Comment by mayed505** [comment]
we could work together im a js dev too i own 24 vps on my home server which was used for my bots :P

**Comment by ghost** [comment]
As the new agar patch  is hard i hired a coder

**Comment by ghost** [comment]
If anyone wanna join team just say
the

**Comment by Nick8fr32** [comment]
i will not have a problem with joining @kurama129

**Comment by ghost** [comment]
Skype of yours ill add you

**Comment by ghost** [comment]
Nick skype

**Comment by Nick8fr32** [comment]
i dont send my skype public email me at: nickniko8132@gmail.com

**Comment by wartoz** [comment]
it's a fake? i see agar.io/?ip.... it's a private server

**Comment by wartoz** [comment]
and you hide the skins shop loading

**Comment by Nick8fr32** [comment]
@Wartoz private servers dont work anymore lol XD

**Comment by wartoz** [comment]
it's easy you took a old screenshot and edit the date

**Comment by wartoz** [comment]
proof send party link

**Comment by wartoz** [comment]
and delete the message

**Comment by ghost** [comment]
Sent

**Comment by Nick8fr32** [comment]
@he didnt he just fixed the server finder thing(easy)

**Comment by wartoz** [comment]
and how you fix the server thing

**Comment by Nick8fr32** [comment]
@kurama129 are u jumbo??

**Comment by ghost** [comment]
Yes but not real and name not based on agario jumbo

**Comment by anonymousxxx02** [comment]
Guys its a scam.... agario bots officially R.I.P

**Comment by mayed505** [comment]
😒 guys just implement agario main_js older into the script instead of requesting it through agario that what raga and client.pw did

**Comment by SrNicolas** [comment]
mayed, talk me trough skype, I got some good things
 spawning bots in party mode, WORKING but I can't press play :p

**Comment by ghost** [comment]
@SrNicolas  I  also make mind spawning but no control over them :( working on new client

**Comment by Nick8fr32** [comment]
@mayed505 what is ur skype

**Comment by mayed505** [comment]
@SrNicolas Sure i could be your meybots.com dev :P i knew about your bot service it looks awesome

**Comment by Yelbert** [comment]
you only need to update the userscript everytime you connect to a new server. I need to find a way to connect the bots AUTOMATICALLY to my server IP like not paste the new party IP to the bots :p I f u can help me with this ill let u work with me.

**Comment by yario-o** [comment]
@Yelbert Update the server automatically is easy but the bots don't work

**Comment by NuclearC** [comment]
nice.

**Comment by yario-o** [comment]
@NuclearC ??

**Comment by ItzLevvie** [comment]
tomato

**Comment by SuperOP535** [comment]
potato

**Comment by NuclearC** [comment]
modulator modulation demodulate module for nodejs

**Comment by SuperOP535** [comment]
?????
